Libraries/Hotcakes.PaypalWebServices/Hotcakes.PaypalWebServices.csproj

```diff
@@ -75,11 +80,22 @@
       <HintPath>..\..\References\Misc\paypal_base.dll</HintPath>
     </Reference>
     <Reference Include="System" />
+    <Reference Include="System.Configuration" />
     <Reference Include="System.ComponentModel.Composition" />
     <Reference Include="System.Core">
       <RequiredTargetFramework>3.5</RequiredTargetFramework>
     </Reference>
     <Reference Include="System.Data" />
+    <Reference Include="System.Runtime.CompilerServices.Unsafe, Version=4.0.4.1, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL">
+      <HintPath>..\..\packages\System.Runtime.CompilerServices.Unsafe.4.5.2\lib\netstandard2.0\System.Runtime.CompilerServices.Unsafe.dll</HintPath>
```

Comment:
This appears to be a new dependency. Was this required by Stripe or something else?

Reply:
Yes. It is required by Stripe